tests/Core/InfoButton/FluentInfoButtonTests.razor

```diff
@@ -0,0 +1,39 @@
+﻿@using Xunit;
+@inherits TestContext
+@code
+{
+    public FluentInfoButtonTests()
+    {
+        JSInterop.Mode = JSRuntimeMode.Loose;
+        Services.AddSingleton(new LibraryConfiguration());
+    }
+
+    [Fact]
+    public void FluentInfoButton_Default()
+    {
+        // Arrange && Act
+        var cut = Render(@<FluentInfoButton>
+        This is example information for an InfoButton
+        <a href="https://www.fluentui-blazor.net/InfoButton">Learn more</a>
+    </FluentInfoButton>);
+
+        // Assert
+        cut.Verify();
+    }
+
+    [Fact]
+    public void FluentInfoButton_AllAttributes()
```

Comment:
Denis meant 1 test per property. For position you can use the `InlineTheory`. Use existing tests for reference.

Reply:
Have you created tests for all individual parameters? If not, do not mark this as resolved then please.